README.md

```diff
@@ -14,10 +14,10 @@ and save it into the labs subdirectory under the name of your lab,
 such as mynewlab.md. It is expected that your lab repository will have
 the same name so keep that in mind.
 
-3. In the Proposal Template, there is an entry for sponsor(s). The sponsor(s) are responsible for reviewing the proposal. Sponsors do not have a responsibility beyond this; ongoing work like contributing code or reviews is not tied to their role as sponsors. In reviewing the proposal, the sponsor(s) make sure that the proposal is cogent, and novel (in conception, proposed execution, or interested community). 
-To find sponsors 
-        a. the proposers can use their connections to existing projects and ask maintainers
-        b. find working groups or projects with affinities to the proposed lab and pitch the project (good to have the template already filled out) in associated channels and or mailing lists. The WG chairs emails, the maintainers contacts etc. can be found on the wiki or github. Make personal appeals if you can.
+3. In the Proposal Template, there is an entry for sponsor(s). Although not required, proposers are encouraged to seek a sponsor who can help them create ties with the rest of the Hyperledger community and ensure that the proposal is cogentand novel (in conception, proposed execution, or interested community).<br/>

```

Comment:
[nitpick 1] rest of the Hyperledger community -> active (and/or) vibrant Hyperledger commuity
[nitpick 2] cogentand -> cogent, and

Reply:
I don't think this is the place for marketing-ish language. ;-) Thanks for spotting the typo.

Reply:
If we are nitpicking, there should be no comma before the and in "cogent and novel". NOTE: This exists two places in the PR.

Reply:
There! :-)